### ISSUE TITLE
[FR]: Alphabet sorting in lists of actions

### ISSUE THREAD

**Issue by sakkamade** [martinrotter/rssguard#449]
#### Brief description of the feature request.

Presently, it is awfully arduous to find a certain function in either Keyboard shortcuts and Available actions of Toolbars lists.
It would be of great help if it had some proper sorting as, for example, sorting in alphabetical order.

**Comment by sakkamade** [comment]
Or perhaps it would be enough to just add some separators with categories of certain actions?

**Comment by sakkamade** [comment]
Never mind alphabetical order, it _is_ in alphabetical order...
I better get some sleep.

**Comment by martinrotter** [comment]
Actions in "keyboard shortcuts" are already sorted:
![image](https://user-images.githubusercontent.com/1255302/124088392-3027fd80-da53-11eb-9ce3-f96884fb22c2.png)

Actions in toolbar editors "available actions" are already sorted:
![image](https://user-images.githubusercontent.com/1255302/124088750-8ac15980-da53-11eb-8b73-21321ef703a1.png)

**Comment by martinrotter** [comment]
As for some "categorization" - yes, this is somewhere quite deep on my internal TODO list.

**Comment by sakkamade** [comment]
> Actions in "keyboard shortcuts" are already sorted:

Yeah: https://github.com/martinrotter/rssguard/issues/449#issuecomment-872018806 ...

**Comment by sakkamade** [comment]
I have slightly revised the request. Sorry, again... :cry:

**Comment by sakkamade** [comment]
> As for some "categorization" - yes, this is somewhere quite deep on my internal TODO list.

Great, but I wonder if it will be consistent with search.
So it would be better to choose only one of them.

Here how the search looks in Akregator:

![akregator filters](https://user-images.githubusercontent.com/62730422/124091273-2a2b2000-da45-11eb-9196-8a1ebab691d8.png)

**Comment by sakkamade** [comment]
So I see few solutions:
1. The categories
    1. The same list, but with separators
       ```
       ---Feed list--------------------------------------
       Action A
       Action B
       Action C
       ---Articles list----------------------------------
       A
       B
       C
       ```
       (^this is how I thought it was but without separators, thence was confused)
      2. Drop down at the top with lists categories 
          ```
           Feeds list                      🔽
          ------------------------------------
          Action A
          Action B
          Action C
          ```
2. The search field https://github.com/martinrotter/rssguard/issues/449#issuecomment-872034385

**Comment by martinrotter** [comment]
This is now de-facto fixed by https://github.com/martinrotter/rssguard/issues/1754 which allows global settings searching which highlights all matching UI elements, including nested ones like toolbar items or keyboard shortcuts.
